crates/astria-composer/src/executor/mod.rs

```diff
@@ -276,7 +287,101 @@ impl Executor {
                     }
                 }
             }
+        };
+
+        match &reason {
+            Ok(reason) => {
+                info!(reason, "shutting down");
+            }
+            Err(reason) => {
+                error!(%reason, "shutting down");
+            }
+        };
+
+        let mut total_bundles_drained_successfully = 0;
+        let shutdown_logic = async {
+            info!("setting executor connection status to disconnected");
+            self.status
+                .send_modify(|status| status.is_connected = false);
+
+            info!("draining sequence actions from the executor receiver channel");
+            // drain the receiver channel
+            while let Ok(seq_action) = self.serialized_rollup_transactions.try_recv() {
+                let rollup_id = seq_action.rollup_id;
+                if let Err(e) = bundle_factory.try_push(seq_action) {
+                    warn!(
+                        rollup_id = %rollup_id,
+                        error = &e as &StdError,
+                        "failed to bundle sequence action: too large. sequence action is dropped."
```

Comment:
```suggestion
                        "failed to bundle sequence action, dropping it"
```
You don't know if the size is the reason - the reason is report by the specific error that `try_push` return, so better to rely on that.